src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -2285,7 +2285,7 @@ int HandlerImpl::emit_deopt_handler(CodeBuffer& cbuf)
 
 //=============================================================================
 
-const bool Matcher::match_rule_supported(int opcode) {
```

Comment:
Have to wonder if these were all meant to be `bool Match:xxx() const {`?

Reply:
Yes, I think that may have been the original intent. I'll add const on these functions.

Reply:
Ah okay :)